SQF/dayz_code/stringtable.xml

```diff
@@ -31751,18 +31821,22 @@
 		<Key ID="STR_EPOCH_SNAP_TUTORIAL_CTRL_PGUPPGDOWN_MSG">
 			<English>Adjust height of object by 1cm.</English>
 			<German>Passt die Höhe des Objektes um 1cm an.</German>
+			<Russian>Настройка высоты (1 см).</Russian>
 		</Key>
 		<Key ID="STR_EPOCH_SNAP_TUTORIAL_ROTATE180_MSG">
 			<English>Rotate object 180 degrees while holding.</English>
 			<German>Dreht das Objekt um 180 Grad.</German>
+			<Russian>Вращение на 180 градусов.</Russian>
 		</Key>
 		<Key ID="STR_EPOCH_SNAP_TUTORIAL_ROTATE45_MSG">
 			<English>Rotate object 45 degrees while holding.</English>
 			<German>Dreht das Objekt um 45 Grad.</German>
+			<Russian>Вращение на 45 градусов.</Russian>
 		</Key>
 		<Key ID="STR_EPOCH_SNAP_TUTORIAL_F_DROP_MSG">
 			<English>Drop / Pick up object.</English>
 			<German>Objekt feststellen / aufnehmen.</German>
+			<Russian>Бросить / Поднять.</Russian>
 		</Key>
 		<Key ID="STR_EPOCH_SNAP_TUTORIAL_F_SNAP_MSG">
 			<English>Drop / Pick up object.</English>
```

Comment:
Is it typo? "STR_EPOCH_SNAP_TUTORIAL_F_DROP_MSG" and "STR_EPOCH_SNAP_TUTORIAL_F_SNAP_MSG" have same english text.

Reply:
Yes you are correct